.idea/btrdb-python.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure we want to commit the .idea files, not sure what the best practice is.

Reply:
agreed. Got past me in the file review.